### ISSUE TITLE
How did you decide on the lengths

### ISSUE THREAD

**Issue by notflip** [JamesClonk/simracing-rig#1]
I'm on the verge of building my own rig, but I'm wondering how do you decide on the lengths?
For example in your design:

2x 1450mm
2x 920mm
1x 600mm

**Comment by JamesClonk** [comment]
@notflip I already owned a rig before (an old version of a Sim-Lab GT1), so I more or less knew what dimensions I'm going to need to for my upgrade, to fit in my room, plus where it needs to be bigger and longer to accomodate the DIY motion actuators.
For example you can also look at the Sim-Lab P1 schematics as a reference / inspiration:
https://sim-lab.eu/download/7862/
https://sim-lab.eu/downloads/
Most Sim-Lab rigs use 600mm width spacing, makes my rig fit nicely for example with their wheel mount accessories:
https://sim-lab.eu/product/front-mount-bracket/
https://sim-lab.eu/product/wheel-deck-bracket/

I've slightly modified and adjusted some components for my rig build in the end (specifically the wheel deck, using a Sim-Lab front mounting, and actuator locations), but it worked out alright in the end:
https://i.imgur.com/CIlRekn.jpg
https://i.imgur.com/OonHZ2H.jpg

**Comment by notflip** [comment]
Your rig looks amazing! Did you add motion yourself? What I see is that Simlab uses their own plates to tilt the pedals, but this is also possible using 80/20 components, right? I've seen it here: https://imgur.com/gallery/TMf3C

What would you recommend for that? I want to keep the price low and customisability high. Thanks for the advice!

**Comment by JamesClonk** [comment]
Yes, it's a DIY motion setup, open source also on github: https://github.com/SimFeedback/SimFeedback-AC-Servo
Check it out, it works great actually! 👍 

The pedal base in my rig design can already be freely tilted. If you check out this picture, it's mounted to the base of the rig with locking levers and angle clamps, so I can move the front or back up and down as I wish:
https://i.imgur.com/7gnI1P3.jpg

I use this lever/clamp set here: https://www.motedis.com/shop/Slot-profile-accessories/Accessoiries-40-I-Type-slot-8/Angle-Clamp-40-with-Locking-Lever-I-Type-slot-8::666.html
Here's their video showing how the clamping mechanism works: https://www.youtube.com/watch?v=2joFMAJI8k8

I'm currently using 6 clamps, 3 per side. But I think 2 per side / 4 total would be perfectly fine too.
The same design is used in Sim-Lab's old GT1 rig for the pedal base: https://sim-lab.eu/download/4419/

**Comment by notflip** [comment]
Aha thank you very much for that! A motion setup is for a new budget round. But thanks for the picture! what does the lever in the middle do? it's hard to see on the picture. Did you base your rig design on other rigs?

**Comment by JamesClonk** [comment]
The middle lever ist just an addition for more stability. It isn't really needed actually.

No, my rig isn't really based on any other design, I just played around in SketchUp until I got something that looked like it would do the job. 😄

**Comment by notflip** [comment]
Does sketchup have extrusion parts included? I'm designing using MayCad but it's so slow! Makes my pc freeze after a while.

**Comment by JamesClonk** [comment]
I remember there being quite lot of different extrusion parts available made by other users on the SketchUp community marketplace (warehouse something it was called I believe).
I got most of the parts needed to model the rig from there.
The aluminum profiles I created myself though, hence their extremely basic look. 😅
